ocs_ci/ocs/ui/block_pool.py

```diff
@@ -173,3 +186,33 @@ def reach_pool_limit(self, replica, compression):
                     raise PoolStateIsUnknow(
                         f"pool {pool_name} is in unexpected state {pool_state}"
                     )
+
+    def pool_raw_capacity_loaded(self, pool_name):
+        """
+        Takes pool name and returns True if the raw capacity of the block pool is loaded
+        or returns False if the capacity is not loaded.
+
+        Args:
+            pool_name (str): The name of the pool to be deleted
+
+        Returns:
+            bool: True if raw capacity of the blockpool is loaded, otherwise False
+        """
+        logger.info("Checking if the Block Pool Raw Capacity has loaded in UI")
+        self.navigate_overview_page()
+        self.navigate_block_pool_page()
+        self.page_has_loaded()
+        if self.ocp_version_full <= version.VERSION_4_8:
+            self.do_click((f"{pool_name}", By.LINK_TEXT))
+        else:
+            self.do_click((f"//a[text()='{pool_name}']", By.XPATH))
```

Comment:
Can you work with the element dictionary.?
https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/ui/views.py
I prefer to add "HTML" code only on views.py file

Reply:
@OdedViner in the above suggestion, the value takes the pool name as a variable value. I think it will be difficult to pass the value of pool name to the element dictionary, as the pool name will vary every time